pyro/infer/reparam/stable.py

```diff
@@ -191,18 +192,18 @@ def __call__(self, name, fn, obs):
         return new_fn, obs
 
 
-def _unsafe_shift(a, skew_abs, t_scale):
+def _unsafe_shift(a, skew, t_scale):
     # At a=1 the lhs has a root and the rhs has an asymptote.
-    return (t_scale - skew_abs) * (math.pi / 2 * a).tan()
+    return (skew.sign() * t_scale - skew) * (math.pi / 2 * a).tan()
```

Comment:
I want to keep the term `skew * tan(a pi / 2)` to preserve grad w.r.t. skew, so I multiplied `skew.sign()` to the previous shift term.